galaxy_ng/app/tasks/synclist.py

```diff
@@ -125,16 +125,21 @@ def curate_synclist_repository(synclist_pk, **kwargs):
 
     namespaces = synclist.namespaces.filter().values_list("name", flat=True)
 
+    # include adds only highest version and exclude removes all versions of the collection.
+    is_highest_query_param = {}
```

Comment:
Explanation for reviewers:

When policy is `exclude` this is going to be an empty dict,so the expansion later will make the query `Q(` to not filter by version including on the queryset all the versions of the collection.

When it is `include` then `is_highest = True` is added to the query, to it filters only the latest version of a collection.